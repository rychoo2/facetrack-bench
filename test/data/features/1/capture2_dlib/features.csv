filename,raw_path,landmark_path,rel_target_x,rel_target_y,timestamp,rel_face_x,rel_face_y,rel_face_size_x,rel_face_size_y,rel_pose_x,rel_pose_y,rel_left_eye_x,rel_left_eye_y,rel_right_eye_x,rel_right_eye_y,rel_left_pupil_face_x,rel_left_pupil_face_y,rel_eye_distance_x,rel_eye_distance_y,rel_right_pupil_face_x,rel_right_pupil_face_y,rel_left_pupil_x,rel_left_pupil_y,rel_right_pupil_x,rel_right_pupil_y
images/20191215123648.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9796834309895833,0.155859375,20191215123648000,0.35546875,-0.06111111111111111,0.2515625,0.49722222222222223,0.5341614906832298,0.7402234636871509,0.515527950310559,0.31564245810055863,0.13975155279503104,0.3407821229050279,0.7142857142857143,0.4134078212290503,0.5496894409937888,-0.030726256983240222,0.30745341614906835,0.441340782122905,0.45714285714285713,0.5303030303030303,0.4153846153846154,0.5454545454545454
images/20191215123629.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.0036878797743055557,0.01906105324074074,20191215123629000,,,,,,,,,,,,,,,,,,,,
images/20191215123639.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.14441460503472223,0.9633391203703704,20191215123639000,,,,,,,,,,,,,,,,,0.5423728813559322,0.3888888888888889,0.4714285714285714,0.5333333333333333
images/20191215123634.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.010639105902777777,0.6308424961419753,20191215123634000,0.4109375,0.0875,0.2515625,0.44722222222222224,0.6024844720496895,0.6055900621118012,0.5217391304347826,0.15217391304347827,0.10869565217391304,0.20186335403726707,0.7484472049689441,0.2236024844720497,0.546583850931677,-0.055900621118012424,0.33540372670807456,0.2795031055900621,0.5748031496062992,0.4339622641509434,0.5069444444444444,0.5102040816326531
images/20191215123635.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.009682888454861111,0.755784625771605,20191215123635000,,,,,,,,,,,,,,,,,0.5423728813559322,0.375,0.4566929133858268,0.48484848484848486
images/20191215123637.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.007859971788194444,0.9523268711419753,20191215123637000,,,,,,,,,,,,,,,,,0.5299145299145299,0.3888888888888889,0.4881889763779528,0.46153846153846156
images/20191215123636.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.007587348090277778,0.9118272569444444,20191215123636000,,,,,,,,,,,,,,,,,,,,
images/20191215123633.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.008323838975694444,0.4863474151234568,20191215123633000,0.4390625,0.0375,0.2515625,0.44722222222222224,0.6366459627329193,0.6987577639751553,0.5559006211180124,0.2919254658385093,0.16770186335403728,0.3198757763975155,0.7360248447204969,0.3136645962732919,0.45962732919254656,-0.031055900621118012,0.36024844720496896,0.37577639751552794,0.6373626373626373,0.1794871794871795,0.4881889763779528,0.46153846153846156
images/20191215123631.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.010767957899305555,0.29573688271604937,20191215123631000,0.4390625,0.0375,0.2515625,0.44722222222222224,0.6459627329192547,0.6583850931677019,0.5590062111801242,0.21739130434782608,0.14596273291925466,0.2453416149068323,0.7391304347826086,0.2608695652173913,0.4782608695652174,-0.027950310559006212,0.36024844720496896,0.3136645962732919,0.6304347826086957,0.30434782608695654,0.5036496350364964,0.4782608695652174
images/20191215123643.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.7875189887152778,0.9573591820987655,20191215123643000,0.38359375,0.0875,0.2515625,0.44722222222222224,0.468944099378882,0.6273291925465838,0.43788819875776397,0.2111801242236025,0.052795031055900624,0.2453416149068323,0.6428571428571429,0.2639751552795031,0.5652173913043478,-0.037267080745341616,0.2267080745341615,0.2950310559006211,0.46153846153846156,0.4722222222222222,0.42748091603053434,0.4444444444444444
images/20191215123642.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.6603339301215277,0.9516541280864198,20191215123642000,0.38359375,0.0875,0.2515625,0.44722222222222224,0.515527950310559,0.6366459627329193,0.4720496894409938,0.20496894409937888,0.062111801242236024,0.2360248447204969,0.6894409937888198,0.2639751552795031,0.577639751552795,-0.031055900621118012,0.2639751552795031,0.30124223602484473,0.48951048951048953,0.475,0.45454545454545453,0.525
images/20191215123640.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.3508992513020833,0.9571566358024691,20191215123640000,0.4109375,0.0875,0.2515625,0.44722222222222224,0.531055900621118,0.6459627329192547,0.45962732919254656,0.2204968944099379,0.055900621118012424,0.2857142857142857,0.6801242236024845,0.2639751552795031,0.5559006211180124,-0.06521739130434782,0.2546583850931677,0.3167701863354037,0.5298507462686567,0.3888888888888889,0.45714285714285713,0.37037037037037035
images/20191215123641.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.47699924045138886,0.9544077932098766,20191215123641000,0.38359375,0.0875,0.2515625,0.44722222222222224,0.5217391304347826,0.6459627329192547,0.4813664596273292,0.2236024844720497,0.08385093167701864,0.2608695652173913,0.7142857142857143,0.2670807453416149,0.5683229813664596,-0.040372670807453416,0.2857142857142857,0.30434782608695654,0.5244755244755245,0.3888888888888889,0.4642857142857143,0.42424242424242425
images/20191215123645.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9801947699652778,0.7978804976851852,20191215123645000,,,,,,,,,,,,,,,,,,,,
images/20191215123651.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.8140584309895833,0.010956790123456791,20191215123651000,0.38359375,0.0375,0.2515625,0.44722222222222224,0.48757763975155277,0.5838509316770186,0.43478260869565216,0.11801242236024845,0.043478260869565216,0.14906832298136646,0.6459627329192547,0.2111801242236025,0.5683229813664596,-0.037267080745341616,0.2329192546583851,0.2391304347826087,0.46258503401360546,0.5084745762711864,0.42657342657342656,0.4915254237288136
images/20191215123650.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9923570421006944,0.026126060956790122,20191215123650000,0.35546875,-0.06111111111111111,0.2515625,0.49722222222222223,0.5372670807453416,0.729050279329609,0.5062111801242236,0.2877094972067039,0.15217391304347827,0.32122905027932963,0.7018633540372671,0.39106145251396646,0.5372670807453416,-0.04189944134078212,0.2981366459627329,0.41899441340782123,0.45,0.5362318840579711,0.3884297520661157,0.5072463768115942
images/20191215123644.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9668429904513889,0.9584104938271605,20191215123644000,0.3890625,0.07222222222222222,0.209375,0.3736111111111111,0.4664179104477612,0.6951672862453532,0.458955223880597,0.29739776951672864,0.07462686567164178,0.3382899628252788,0.7089552238805971,0.3382899628252788,0.6007462686567164,-0.048327137546468404,0.3283582089552239,0.38661710037174724,0.48905109489051096,0.3333333333333333,0.576271186440678,0.43333333333333335
images/20191215123652.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,,,,0.38359375,0.0375,0.2515625,0.44722222222222224,0.5372670807453416,0.5869565217391305,0.484472049689441,0.09316770186335403,0.07763975155279502,0.12732919254658384,0.6987577639751553,0.19875776397515527,0.5683229813664596,-0.040372670807453416,0.27639751552795033,0.2360248447204969,0.4928571428571429,0.5151515151515151,0.4444444444444444,0.5303030303030303
images/20191215123646.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.98377685546875,0.6164086612654321,20191215123646000,,,,,,,,,,,,,,,,,0.5470085470085471,0.3055555555555556,0.3162393162393162,0.30303030303030304
images/20191215123647.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9801201714409722,0.29488088348765434,20191215123647000,0.38359375,0.0375,0.2515625,0.44722222222222224,0.43167701863354035,0.6273291925465838,0.41304347826086957,0.17391304347826086,0.046583850931677016,0.2111801242236025,0.6055900621118012,0.2639751552795031,0.5434782608695652,-0.043478260869565216,0.20186335403726707,0.2919254658385093,0.44285714285714284,0.5178571428571429,0.3937007874015748,0.49056603773584906
