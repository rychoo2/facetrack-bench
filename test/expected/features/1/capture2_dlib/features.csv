filename,raw_path,landmark_path,rel_target_x,rel_target_y,timestamp,rel_face_x,rel_face_y,rel_face_size_x,rel_face_size_y,rel_pose_x,rel_pose_y,rel_left_eye_x,rel_left_eye_y,rel_right_eye_x,rel_right_eye_y,rel_left_pupil_face_x,rel_left_pupil_face_y,rel_eye_distance_x,rel_eye_distance_y,rel_right_pupil_face_x,rel_right_pupil_face_y,rel_left_pupil_x,rel_left_pupil_y,rel_right_pupil_x,rel_right_pupil_y
images/20191215123648.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9796834309895833,0.155859375,20191215123648000,0.35546875,-0.06111111111111111,0.2515625,0.49722222222222223,0.5434782608695652,0.7402234636871509,0.562111801242236,0.34916201117318435,0.18012422360248448,0.38268156424581007,0.7111801242236024,0.4134078212290503,0.5496894409937888,-0.05027932960893855,0.30745341614906835,0.441340782122905,0.4444444444444444,0.5111111111111111,0.3942307692307692,0.4666666666666667
images/20191215123629.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.0036878797743055557,0.01906105324074074,20191215123629000,,,,,,,,,,,,,,,,,,,,
images/20191215123639.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.14441460503472223,0.9633391203703704,20191215123639000,0.4109375,0.0875,0.2515625,0.44722222222222224,0.6211180124223602,0.639751552795031,0.5900621118012422,0.2329192546583851,0.18633540372670807,0.2950310559006211,0.7515527950310559,0.2577639751552795,0.5341614906832298,-0.07142857142857142,0.34782608695652173,0.3198757763975155,0.5591397849462365,0.32,0.47706422018348627,0.4
images/20191215123634.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.010639105902777777,0.6308424961419753,20191215123634000,0.4109375,0.0875,0.2515625,0.44722222222222224,0.5869565217391305,0.6086956521739131,0.562111801242236,0.17701863354037267,0.14285714285714285,0.22981366459627328,0.7484472049689441,0.2236024844720497,0.5590062111801242,-0.055900621118012424,0.3385093167701863,0.2826086956521739,0.594059405940594,0.4166666666666667,0.525,0.53125
images/20191215123635.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.009682888454861111,0.755784625771605,20191215123635000,0.4390625,0.0875,0.2515625,0.44722222222222224,0.5279503105590062,0.5962732919254659,0.48757763975155277,0.20186335403726707,0.08385093167701864,0.2360248447204969,0.6614906832298136,0.21739130434782608,0.5217391304347826,-0.043478260869565216,0.2546583850931677,0.2732919254658385,0.6363636363636364,0.2,0.5092592592592593,0.48
images/20191215123637.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.007859971788194444,0.9523268711419753,20191215123637000,0.4390625,0.0875,0.2515625,0.44722222222222224,0.5496894409937888,0.6149068322981367,0.5062111801242236,0.20496894409937888,0.10559006211180125,0.2515527950310559,0.6708074534161491,0.2236024844720497,0.5279503105590062,-0.059006211180124224,0.2732919254658385,0.2857142857142857,0.5760869565217391,0.25,0.5,0.4583333333333333
images/20191215123636.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.007587348090277778,0.9118272569444444,20191215123636000,0.4390625,0.0875,0.2515625,0.44722222222222224,0.5434782608695652,0.6242236024844721,0.5062111801242236,0.20496894409937888,0.10559006211180125,0.2577639751552795,0.6677018633540373,0.2360248447204969,0.5279503105590062,-0.059006211180124224,0.2701863354037267,0.2950310559006211,0.5652173913043478,0.35714285714285715,0.49074074074074076,0.48
images/20191215123633.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.008323838975694444,0.4863474151234568,20191215123633000,0.4390625,0.0375,0.2515625,0.44722222222222224,0.6335403726708074,0.6956521739130435,0.5900621118012422,0.27639751552795033,0.19254658385093168,0.32608695652173914,0.7391304347826086,0.3136645962732919,0.4937888198757764,-0.055900621118012424,0.35714285714285715,0.37267080745341613,0.6233766233766234,0.3333333333333333,0.5047619047619047,0.46875
images/20191215123631.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.010767957899305555,0.29573688271604937,20191215123631000,0.4390625,0.0375,0.2515625,0.44722222222222224,0.639751552795031,0.6583850931677019,0.5838509316770186,0.2360248447204969,0.19875776397515527,0.2546583850931677,0.7391304347826086,0.2639751552795031,0.4813664596273292,-0.024844720496894408,0.36024844720496896,0.3136645962732919,0.6493506493506493,0.25,0.5,0.475
images/20191215123643.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.7875189887152778,0.9573591820987655,20191215123643000,0.38359375,0.0875,0.2515625,0.44722222222222224,0.468944099378882,0.6273291925465838,0.48757763975155277,0.2236024844720497,0.08074534161490683,0.2608695652173913,0.6428571428571429,0.2608695652173913,0.5838509316770186,-0.043478260869565216,0.2236024844720497,0.2950310559006211,0.4424778761061947,0.42857142857142855,0.42201834862385323,0.44
images/20191215123642.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.6603339301215277,0.9516541280864198,20191215123642000,0.38359375,0.0875,0.2515625,0.44722222222222224,0.5124223602484472,0.6366459627329193,0.5217391304347826,0.2111801242236025,0.09627329192546584,0.2515527950310559,0.6894409937888198,0.2608695652173913,0.5962732919254659,-0.043478260869565216,0.2639751552795031,0.2981366459627329,0.4778761061946903,0.5,0.46551724137931033,0.5172413793103449
images/20191215123640.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.3508992513020833,0.9571566358024691,20191215123640000,0.43515625,0.15555555555555556,0.21015625,0.37222222222222223,0.5092936802973977,0.5895522388059702,0.49070631970260226,0.1044776119402985,0.04460966542750929,0.1791044776119403,0.6988847583643123,0.13432835820895522,0.620817843866171,-0.08582089552238806,0.19330855018587362,0.20149253731343283,0.5773195876288659,0.27586206896551724,0.4,0.2857142857142857
images/20191215123641.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.47699924045138886,0.9544077932098766,20191215123641000,0.38359375,0.0875,0.2515625,0.44722222222222224,0.5279503105590062,0.639751552795031,0.5341614906832298,0.2236024844720497,0.12422360248447205,0.2670807453416149,0.7111801242236024,0.2670807453416149,0.5838509316770186,-0.052795031055900624,0.2857142857142857,0.30434782608695654,0.504424778761062,0.4827586206896552,0.46017699115044247,0.42857142857142855
images/20191215123645.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9801947699652778,0.7978804976851852,20191215123645000,0.35546875,0.0375,0.2515625,0.44722222222222224,0.546583850931677,0.717391304347826,0.5683229813664596,0.2795031055900621,0.17080745341614906,0.32298136645962733,0.7080745341614907,0.3416149068322981,0.5683229813664596,-0.049689440993788817,0.39751552795031053,0.38509316770186336,0.41284403669724773,0.5555555555555556,0.6952380952380952,0.6060606060606061
images/20191215123651.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.8140584309895833,0.010956790123456791,20191215123651000,0.38359375,0.0375,0.2515625,0.44722222222222224,0.48757763975155277,0.5838509316770186,0.4968944099378882,0.14285714285714285,0.09937888198757763,0.17391304347826086,0.6490683229813664,0.2080745341614907,0.5652173913043478,-0.043478260869565216,0.2329192546583851,0.2422360248447205,0.44954128440366975,0.4772727272727273,0.3944954128440367,0.4888888888888889
images/20191215123650.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9923570421006944,0.026126060956790122,20191215123650000,0.35546875,-0.06111111111111111,0.2515625,0.49722222222222223,0.5372670807453416,0.729050279329609,0.5590062111801242,0.31843575418994413,0.18012422360248448,0.3575418994413408,0.7049689440993789,0.39106145251396646,0.5434782608695652,-0.05027932960893855,0.2981366459627329,0.42178770949720673,0.44761904761904764,0.5306122448979592,0.38,0.4791666666666667
images/20191215123644.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9668429904513889,0.9584104938271605,20191215123644000,0.3890625,0.07222222222222222,0.209375,0.3736111111111111,0.4626865671641791,0.7137546468401487,0.5149253731343284,0.31226765799256506,0.12313432835820895,0.34944237918215615,0.7052238805970149,0.3345724907063197,0.5895522388059702,-0.04460966542750929,0.3283582089552239,0.38661710037174724,0.504950495049505,0.24,0.625,0.4166666666666667
images/20191215123652.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,,,,0.38359375,0.0375,0.2515625,0.44722222222222224,0.5372670807453416,0.5838509316770186,0.5341614906832298,0.14285714285714285,0.13043478260869565,0.18633540372670807,0.6987577639751553,0.20186335403726707,0.5683229813664596,-0.046583850931677016,0.2795031055900621,0.2360248447204969,0.49074074074074076,0.475,0.44036697247706424,0.43243243243243246
images/20191215123646.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.98377685546875,0.6164086612654321,20191215123646000,0.3890625,0.07222222222222222,0.209375,0.3736111111111111,0.5,0.7026022304832714,0.5261194029850746,0.18587360594795538,0.08582089552238806,0.23048327137546468,0.7052238805970149,0.26022304832713755,0.6417910447761194,-0.04460966542750929,0.21641791044776118,0.2936802973977695,0.45714285714285713,0.5,0.36082474226804123,0.4594594594594595
images/20191215123647.jpg,data/raw/capture2/positions.csv,data/landmarks/1/capture2/landmarks.csv,0.9801201714409722,0.29488088348765434,20191215123647000,0.3890625,0.07222222222222222,0.209375,0.3736111111111111,0.48880597014925375,0.6579925650557621,0.5186567164179104,0.1524163568773234,0.07462686567164178,0.19330855018587362,0.7014925373134329,0.22304832713754646,0.6529850746268657,-0.05947955390334572,0.21641791044776118,0.25650557620817843,0.4537037037037037,0.475,0.3917525773195876,0.425
